File: College-Sports-WebApp/Database/Models/ViewingPolicy.cs
using System.Collections.Generic; 
namespace College_Sports_WebApp.Database.Models{ 

    public class ViewingPolicy
    {
        public Audience audience { get; set; }
        public List<string> actions { get; set; }
        public string name { get; set; }
        public string id { get; set; }
        public string externalId { get; set; }
    }

}

File: College-Sports-WebApp/ClientApp/src/api/models/viewing-policy.ts
/* tslint:disable */
/* eslint-disable */
import { Audience } from '../models/audience';
export interface ViewingPolicy {
  actions?: Array<string> | null;
  audience?: Audience;
  externalId?: string | null;
  id?: string | null;
  name?: string | null;
}
